lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -32,6 +32,8 @@ patch.facecolor        : b
 patch.edgecolor        : k
 patch.antialiased      : True    # render patches in antialiased (no jaggies)
 
+hatch.linewidth        : 1.0
```

Comment:
Actually 4, since SVG was "1 point" and Agg was "1 pixel".  We could try to emulate that old brokenness if we want (I guess).